artifact/artifact/v1alpha/artifact.proto

```diff
@@ -144,10 +144,23 @@ message KnowledgeBase {
   string name = 2;
   // The knowledge base description.
   string description = 3;
+  // Whether the knowledge base is in use.
+  bool inuse = 4;
```

Comment:
```suggestion
  bool in_use = 4;
```

Can you share some use cases of `inuse`?

Reply:
it probably will be a controller or a indicator whether this kb is used moved out due to not be included in this scope.